ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/IndentationRule.kt

```diff
@@ -673,16 +674,6 @@ class IndentationRule : Rule("indent"), Rule.Modifier.RestrictToRootLast {
         if (!ctx.ignored.contains(p) && nextSibling != null) {
             expectedIndent++
             debug { "++inside(${p.elementType}) -> $expectedIndent" }
-            val siblingType = nextSibling.elementType
```

Comment:
This is just a cleanup, right?

Reply:
Yes, that is right. I committed this separately with message `remove unused variables`.